scipy/stats/tests/test_continuous.py

```diff
@@ -1120,7 +1120,7 @@ def parameters(self):
 
             @property
             def support(self):
-                return 'a', 'b'
+                return {'endpoints': ('a', 'b'), 'inclusive': (True, True)}
```

Comment:
```suggestion
                return {'endpoints': ('a', 'b')}
```
We could test the default here.